rlclientlib/logger/logger_facade.cc

```diff
@@ -351,5 +351,42 @@ int observation_logger_facade::report_action_taken(const char* primary_id, const
       return protocol_not_supported(status);
   }
 }
+
+
+//TODO: Do we need an EPISODE_SECTION for the config? Just use OBSERVATION_SECTION for now
```

Comment:
The config for Episode were incomplete comparing with interaction/observation: https://github.com/VowpalWabbit/reinforcement_learning/blob/b7b33f8f52634644ebf686c5427058ef1132174d/include/constants.h#L22
The missing ones are used here so we'll need to complete those configs first if we don't want to reuse observation.